### PR TITLE
feat: Setup NTP

### PR DESCRIPTION
**Pull request by samuelarogbonlo**
**Summary of changes**
Changes introduced in this pull request:
- Install and Enable NTP to synchronize the time differences

**Reference issue to close (if applicable)**

Closes 

**Other information and links**

### CODE REVIEW

**Review by aatifsyed — APPROVED** [review]

**Review by LesnyRumcajs — CHANGES_REQUESTED** [review]
I don't quite understand. You reverted NTP changes in https://github.com/ChainSafe/forest-iac/pull/146. Could you please provide context for both reverting and bringing it back again?

**Comment by LesnyRumcajs** [comment]
Additionally, this branch contains changes from https://github.com/ChainSafe/forest-iac/pull/219, which should not be the case.

File: .github/workflows/deploy-sync-check.yml [hunk 1]
```diff
@@ -35,3 +35,4 @@ jobs:
           working_directory: terraform/sync_check
           environment: Sync Check Service
           nr_license_key: ${{ secrets.NR_LICENSE_KEY }}
+
```
Comment (LesnyRumcajs) [inline .github/workflows/deploy-sync-check.yml:38]:
```suggestion
```

File: terraform/modules/filecoin_node/firewall.tf [hunk 2]
```diff
@@ -31,6 +31,11 @@ resource "digitalocean_firewall" "forest_firewall" {
     destination_addresses = var.destination_addresses
   }
 
+  outbound_rule {
```
Comment (LesnyRumcajs) [inline terraform/modules/filecoin_node/firewall.tf:34]: Let's add a comment on why it's needed.

**Pull request merged**